src/server/vfs/mac/litesyncextconnector.mm

```diff
@@ -1128,7 +1128,7 @@ bool checkIoErrorAndLogIfNeeded(IoError ioError, const std::string &itemType, co
                         }
 
                         if (![fileHandle writeData:buffer error:&error]) {
-                            LOGW_ERROR(_logger, L"Error while writting to file - path=" << QStr2WStr(filePath).c_str()
+                            LOGW_ERROR(_logger, L"Error while writing to file - path=" << QStr2WStr(filePath).c_str()
```

Comment:
```suggestion
                            LOGW_ERROR(_logger, L"Error while writing to: " << Utility::formatSyncPath(filePath).c_str()
```
If there is no dependency issue preventing us from using `KDC::Utility` here.

Reply:
Since 'filePath' is a 'QString' here, I would suggest to keep it like that until the refactoring that will remove Qt everywhere